frontend/public/locales/en/translation.json

```
@@ -2783,6 +2786,8 @@
     "The existing namespace where the credential secret will be stored.": "The existing namespace where the credential secret will be stored.",
     "The following Argo application(s) deployed by the application set will also be deleted:": "The following Argo application(s) deployed by the application set will also be deleted:",
     "The fully-qualified host name or IP address of the vCenter server. The value must be defined in the vCenter server root CA certificate.": "The fully-qualified host name or IP address of the vCenter server. The value must be defined in the vCenter server root CA certificate.",
+    "The Globalnet CIDR to be used for the  managed cluster (If left blank CIDR will be allocated from clusterset pool).": "The Globalnet CIDR to be used for the  managed cluster (If left blank CIDR will be allocated from clusterset pool).",
+    "The globalnet CIDR to be used. The default is 242.0.0.0/8": "The globalnet CIDR to be used. The default is 242.0.0.0/8",
```

As we are making reference to clusterset pool during the join operation, lets use it here to convey that it points to this value.

```suggestion
    "The Globalnet CIDR to be used within the clusterset pool. The default is 242.0.0.0/8": "The globalnet CIDR to be used within the clusterset pool. The default is 242.0.0.0/8",
```